server/src/main/java/com/linecorp/centraldogma/server/internal/api/TokenLevelRequest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2024 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.centraldogma.server.internal.api;
+
+class TokenLevelRequest {
+    private String level;
+
+    TokenLevelRequest() {
+    }
+
+    TokenLevelRequest(String level) {
+        this.level = level;
+    }
+
+    String getLevel() {
+        return level;
+    }
+
+    void setLevel(String level) {
+        this.level = level;
+    }
+}
```

Comment:
Let's use Jackson annotations to make it clear. 😉 
```suggestion
class TokenLevelRequest {

    private final String level;

    @JsonCreator
    TokenLevelRequest(@JsonProperty("level") String level) {
        this.level = level;
    }

    @JsonProperty
    String level() {
        return level;
    }
}
```